README.md

```diff
@@ -6,45 +6,45 @@
 
 # Make your own fork of this app in Codespaces
 
-Step 1:  Click the "Open in Github Codespaces" button above and then click Fork!
+Step 1:  Right click the "Open in Github Codespaces" button above, open the link in a new tab and then click Fork!
 
 ![image](https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/f4a9d972-cae6-4aa2-8b9a-2e18f513759a)
 
-_(Eat a cookie while you wait)_
+_(Eat a 🍪 while you wait)_
 
 Step 2: Press "Create new Codespace"
 
 <img width="521" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/87d24ecc-daf2-4292-a0fa-13e6fc68094e">
 
 _(Check your bluesky while you wait)_
 
-Step 3: After a minute or two, you should see an editor, and then a browser should open to the right (You can resize the site to see the whole app better)
+Step 3: After a minute or two, you should see an editor (it may take a few seconds after the page has loaded), and then a browser should open to the right (You can resize the site to see the whole app better)
 
 <img width="1684" alt="image" src="https://github.com/blackary/streamlit-vizzu-pydata-global/assets/4040678/04f1be56-7b71-4303-81f0-ed7679e4b1a0">
 
 # Add your own charts
 
-Step 1: Download one of the csvs in this repository
+Step 1: Download one of the csvs above in this repository
 
-Step 2: Upload it to https://vizzu-builder.streamlit.app/
+Step 2: In a new tab, open https://vizzu-builder.streamlit.app/ and upload the CSV you just downloaded.
 
-Step 3: Choose 1 Category and one or more Values columns and hit Update Charts to see the different kinds of charts you can make
+Step 3: Choose 1 Category and one or more Values columns and hit `Create Charts` to see the different kinds of charts you can make.
```

Comment:
I think you should rename this in the app -- update charts to me means they already exist.

Reply:
Yeah, that's a good call. Do you think 'create charts' is confusing when they already exist? If you upload a new dataset, or change the filter, should it always say "Create charts"? Or should I use session state or something to make it first say "Create charts", and then "update charts"?